TransformationSystem/Client/TaskManager.py

```diff
@@ -559,7 +559,7 @@ def __prepareTasksBulk(self, transBody, taskDict, owner, ownerGroup, ownerDN):
     oJob.setJobGroup(transGroup)
 
     if int(transID) in [int(x) for x in self.opsH.getValue("Hospital/Transformations", [])]:
-      self._handleHospital(oJob)
+      self._handleHospital(oJob,transID)
```

Comment:
The tests will also fail due to the missing space here and on the instance of _handleHospital
```suggestion
      self._handleHospital(oJob, transID)
```